torch/fx/experimental/symbolic_shapes.py

```diff
@@ -26,6 +26,17 @@
     from sympy.printing.precedence import precedence  # type: ignore[import] # noqa: F401
     from sympy.printing.str import StrPrinter  # type: ignore[import]
     HAS_SYMPY = True
+
+    original_expand = sympy.expand
```

Comment:
note - this is a micro optimization on top of a bigger one, the lru_cache. We can make this PR simpler at a cost of a little perf by only caching our safe_expand instead of applying it to all sympy expand.